guide/content/deployment-steps.md

```diff
@@ -13,13 +13,15 @@ description: Deployment steps
 * **PrincipalArn**: `arn:aws:iam::123456789012:root` (Amazon Resource Name of the principal that can assume the role.
 * **pEnableCloudTrial**: `true` (Enables CloudTrail at the organization level. Default is false.)
 * **pEnableGuardDuty**: `true` (Enables GuardDuty at the organization level. Default is false.)
+* **pDisableGuardDuty**: `Yes` (Disable the GuardDuty solution in all accounts and regions before deleting the stack. Default is false.)
```

Comment:
The description of the field isn't really clear to me.

Is this a deletion that happens IF someone deletes the stack later?

Right now we have 2 confusing parameters after each other.

pEnableGuardDuty followed by pDisableGuardDuty

Reply:
Replaced `pEnableGuardDuty` with `pConfigureGuardDuty` and `pEnableCloudTrail` with `pConfigureCloudTrail`.  These parameters meant to prompt the customer if these services need to be configured in the account are not (these are specific to this ABI).

The `pDisableGuardDuty` is a new parameter added to the SRA module to allow customers to choose if they want to completely disable the GuardDuty when they delete the solution. The default is set to `No`, as customers may need to retain the findings in cases where this solution was running for a long duration and customer want to retain the findings in the GuardDuty service for business reasons.  There are additional cleanup steps needed, if customers choose the default value `pDisableGuardDuty: No`. Will update the cleanup section.